.gitmodules

```diff
@@ -6,4 +6,4 @@
 	url = https://github.com/ocurrent/ocurrent
 [submodule "vendor/current-web-pipelines"]
 	path = vendor/current-web-pipelines
-	url = https://github.com/TheLortex/current-web-pipelines.git
+	url = https://github.com/benmandrew/current-web-pipelines.git
```

Comment:
Please make a PR for these changes to [tarides/current-web-pipelines](https://github.com/tarides/current-web-pipelines).